src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbData/breadcrumbDataException.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import omit from 'lodash/omit';
+
+import {getMeta} from 'app/components/events/meta/metaProxy';
+import {defined} from 'app/utils';
+
+import getBreadcrumbCustomRendererValue from '../../breadcrumbs/getBreadcrumbCustomRendererValue';
+import {BreadcrumbTypeDefault} from '../../breadcrumbs/types';
+import BreadcrumbDataSummary from './breadcrumbDataSummary';
+
+type Props = {
+  breadcrumb: BreadcrumbTypeDefault;
+};
+
+const BreadcrumbDataException = ({breadcrumb}: Props) => {
+  const {data} = breadcrumb;
+  return (
+    <BreadcrumbDataSummary kvData={omit(data, ['type', 'value'])}>
+      {data?.type &&
+        getBreadcrumbCustomRendererValue({
+          value: <strong>{`${data.type}: `}</strong>,
+          meta: getMeta(data, 'type'),
+        })}
+      {defined(data) &&
+        defined(data?.value) &&
```

Comment:
Should we merge this?

Reply:
I have updated it...I believe now it's better...but the reason I did like above is that `data` can be `undefined` or `null` and `value` can be `0 `or something else